src/routes/edit.svelte

```diff
@@ -145,6 +164,11 @@
 									bind:checked={$codeStore.autoSync} />
 								<span> Auto sync</span>
 							</label>
+							<a
+								target="_blank"
+								class="btn btn-primary btn-xs shadow-lg"
+								href="https://mermaid-js.github.io/mermaid/{docMap[docKey] || ''}"
+								><i class="far fa-eye mr-2" /> DOC</a>
```

Comment:
<img width="235" alt="image" src="https://user-images.githubusercontent.com/10703445/157647128-55b0ed8d-17c2-4906-ab66-c06bae8da701.png">

Can we use a different icon for docs? 
It's not very evident that it's a button as there's auto sync also there.

Reply:
![image](https://user-images.githubusercontent.com/3371163/157668991-aa2bb879-e9a4-4586-9dff-10be13728983.png)
![image](https://user-images.githubusercontent.com/3371163/157669029-31e550b0-3be9-4e7f-a29d-7d602d5d59a0.png)
how about this?

Reply:
![image](https://user-images.githubusercontent.com/3371163/157678549-128f09e3-1b27-40e0-bece-46859ba3a47c.png)
![image](https://user-images.githubusercontent.com/3371163/157678585-46e5c4bd-cd43-4678-804a-3a94745c85f2.png)
Or like this?

Reply:
This is using `btn-secondary` right?
Can you also change the view button on the right to same?
And a different icon than eye would be great.

Reply:
![image](https://user-images.githubusercontent.com/3371163/157682656-6438c31a-2233-44f5-a5c0-eea97206d942.png)
![image](https://user-images.githubusercontent.com/3371163/157682723-56148a71-01c5-4bf2-9c4e-1c271e83b84c.png)
![image](https://user-images.githubusercontent.com/3371163/157682793-a65ab727-8926-4d17-b3d8-35c25315dcfd.png)

Reply:
> This is using `btn-secondary` right? Can you also change the view button on the right to same? And a different icon than eye would be great.

yes，`btn-secondary` is used